tests/builders/test_response_builder.py

```diff
@@ -44,7 +47,9 @@ def _get_builder_mock(expected_value: Any) -> Union[ContentBuilder, HeaderBuilde
                 "application/json": {
                     "schema": {
                         "type": "string",
-                    }
+                    },
+                    "example": "an example",
+                    "examples": ["example1", "example2"]
```

Comment:
However, I already posted a few comments on your #88 PR about the type of the `examples` field, according to the spec it is not allowed to have `list`